### PR TITLE
feat(react-ui-kit): added CheckboxLabel component, updated button style and font sizes

### PR DESCRIPTION
**Pull request by AndyLnd**

### CODE REVIEW

File: packages/react-ui-kit/src/Form/Checkbox.jsx [hunk 1]
```diff
@@ -40,24 +43,42 @@ const StyledCheckbox = Input.withComponent('input').extend.attrs({
   margin-bottom: 0;
 `;
 
-const Checkbox = ({id, children, ...props}) => {
+const Checkbox = ({id, children, style, ...props}) => {
+  if (id === null) {
```
Comment (lipis) [inline packages/react-ui-kit/src/Form/Checkbox.jsx]: `if (!id) {` ...

File: packages/react-ui-kit/src/Form/Checkbox.jsx [hunk 2]
```diff
@@ -17,10 +17,13 @@
  *
  */
 
+import {COLOR} from '../index';
 import {Input} from './Input';
 import PropTypes from 'prop-types';
 import React from 'react';
+import {Text} from '../Text';
 import styled from 'styled-components';
+import {v4 as uuid} from 'uuid';
```
Comment (bennyn) [inline packages/react-ui-kit/src/Form/Checkbox.jsx]: Do you really need a `uuid` in a styled component? To me UUID sounds like business logic. Nothing that you would want to important in a presentation layer.

File: packages/react-ui-kit/src/Form/CodeInput.jsx [hunk 3]
```diff
@@ -126,6 +131,7 @@ class CodeInput extends React.PureComponent {
     for (let i = 0; i < this.props.digits; i++) {
       inputs.push(
         <DigitInput
+          autoFocus={i === 0 && this.props.autoFocus}
```
Comment (lipis) [inline packages/react-ui-kit/src/Form/CodeInput.jsx]: `{i && this.props.autoFocus}`

File: packages/react-ui-kit/src/Form/Checkbox.jsx (hunk 1 above)
Comment (AndyLnd) [reply packages/react-ui-kit/src/Form/Checkbox.jsx]: actually i really want to check against null, as this is the default value i set.

File: packages/react-ui-kit/src/Form/CodeInput.jsx (hunk 3 above)
Comment (lipis) [inline packages/react-ui-kit/src/Form/CodeInput.jsx]: `{!i && this.props.autoFocus}`

File: packages/react-ui-kit/src/Form/CodeInput.jsx [hunk 4]
```diff
@@ -126,6 +131,7 @@ class CodeInput extends React.PureComponent {
     for (let i = 0; i < this.props.digits; i++) {
```
Comment (lipis) [inline packages/react-ui-kit/src/Form/CodeInput.jsx]: `index` instead of `i`

File: packages/react-ui-kit/src/Form/Checkbox.jsx (hunk 1 above)
Comment (lipis) [reply packages/react-ui-kit/src/Form/Checkbox.jsx]: oki..

File: packages/react-ui-kit/src/Form/CodeInput.jsx [hunk 5]
```diff
@@ -123,21 +128,22 @@ class CodeInput extends React.PureComponent {
...
-          onChange={e => this.setValue(i, e.target.value)}
-          onPaste={e => this.handlePaste(i, e.clipboardData.getData('Text'))}
+          autoFocus={index === 0 && this.props.autoFocus}
+          key={index}
+          onChange={e => this.setValue(index, e.target.value)}
+          onPaste={e => this.handlePaste(index, e.clipboardData.getData('Text'))}
           onFocus={e => e.target.select()}
-          onKeyDown={e => this.handleKeyDown(i, e)}
-          innerRef={node => (this.inputs[i] = node)}
+          onKeyDown={e => this.handleKeyDown(index, e)}
```
Comment (lipis) [inline packages/react-ui-kit/src/Form/CodeInput.jsx]: `e` -> `error`

Comment (lipis) [reply packages/react-ui-kit/src/Form/CodeInput.jsx]: https://github.com/wireapp/wire-web-packages/pull/47

File: packages/react-ui-kit/src/Form/Checkbox.jsx (hunk 2 above)
Comment (AndyLnd) [reply packages/react-ui-kit/src/Form/Checkbox.jsx]: Maybe not. Actually I just need some random string. Something like Math.random().toString() would work as well.

File: packages/react-ui-kit/src/Form/Checkbox.jsx (hunk 1 above)
Comment (AndyLnd) [reply packages/react-ui-kit/src/Form/Checkbox.jsx]: but yeah, as every falsy value would not be fit as a key, i am going to change that.

File: packages/react-ui-kit/src/Form/Checkbox.jsx (hunk 2 above)
Comment (bennyn) [reply packages/react-ui-kit/src/Form/Checkbox.jsx]: 👍 , then we can also remove 'uuid' from the dependencies?

**Pull request merged**